src/common/primitive_serialization.hpp

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+* Copyright 2021-2025 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef COMMON_PRIMITIVE_SERIALIZATION_HPP
+#define COMMON_PRIMITIVE_SERIALIZATION_HPP
+
+#include "common/c_types_map.hpp"
+#include "common/primitive_attr.hpp"
+#include "common/serialization.hpp"
+#include "common/type_helpers.hpp"
+#include "oneapi/dnnl/dnnl.h"
```

Comment:
Minor: probably can skip this one in favor of c_types_map.

Reply:
Removed, thanks.